folly/FBString.h

```diff
@@ -98,13 +99,7 @@ FOLLY_GNU_DISABLE_WARNING("-Wshadow")
 
 FOLLY_FBSTRING_BEGIN_NAMESPACE
 
-#if defined(__clang__)
-#if __has_feature(address_sanitizer)
-#define FBSTRING_SANITIZE_ADDRESS
-#endif
-#elif defined(__GNUC__) &&                                             \
-    (((__GNUC__ == 4) && (__GNUC_MINOR__ >= 8)) || (__GNUC__ >= 5)) && \
-    __SANITIZE_ADDRESS__
+#if defined(FOLLY_SANITIZE_ADDRESS) && defined(__GNUC__)
```

Comment:
Don't need the `__GNUC__` test anymore, then.

Which implies that we don't need `FBSTRING_SANITIZE_ADDRESS` as a separate symbol anymore and uses may be replaced with `FOLLY_SANITIZE_ADDRESS`.

Reply:
Good point; even better! Just made that change. 👍